GitUI/ScriptsEngine/IRunScript.cs

```diff
@@ -4,5 +4,14 @@
     public interface IRunScript
```

Comment:
As per the comment - this interface will be removed, so it shouldn't be used.

Reply:
I can only remove the use thereof once a replacement is created. The refactor, as done so far, has not removed all uses of `IRunScript`, and has not produced an alternative. Best I can do is rebase to whatever alternative is offered by any part 2 of the refactor, if one such is merged before this PR.